pkg/lib/oauth/handler/handler_authz.go

```diff
@@ -26,9 +26,10 @@ import (
 //go:generate mockgen -source=handler_authz.go -destination=handler_authz_mock_test.go -package handler_test
 
 const (
-	CodeResponseType          = "code"
-	NoneResponseType          = "none"
-	SettingsActonResponseType = "urn:authgear:params:oauth:response-type:settings-action"
+	CodeResponseType                      = "code"
+	NoneResponseType                      = "none"
+	SettingsActonResponseType             = "urn:authgear:params:oauth:response-type:settings-action"
+	AppInitiatedSSOToWebTokenResponseType = "urn:authgear:params:oauth:response-type:app_initiated_sso_to_web token"
```

Comment:
I think we should define AppInitiatedSSOToWebTokenResponseType as "urn:authgear:params:oauth:response-type:app_initiated_sso_to_web" and then parse the response type as an unordered list of string. if the unordered list of string is ["urn:authgear:params:oauth:response-type:app_initiated_sso_to_web", "token"] then it is the request we want to handle. Otherwise it is invalid response type.